src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -243,6 +244,10 @@ private struct Entry {
         }
 
         public bool ContainsValue(TValue value) {
```

Comment:
@mikedn You're probably right, but it's only +2 lines of code and a relatively small method so I don't see why not.